hs_core/templates/pages/genericresource.html

```diff
@@ -772,9 +773,9 @@ <h4 class="modal-title">Manage access</h4>
 
                                                 <ul id="list-roles" class="dropdown-menu"
                                                     aria-labelledby="roles_list">
-                                                    <li><a data-role="view">Can view</a></li>
-                                                    <li><a data-role="edit">Can edit</a></li>
-                                                    <li><a data-role="owner">Is owner</a></li>
+                                                    <li {% if not is_view_user and not is_edit_user and not is_owner_user %}class="disabled"{% endif %}><a data-role="view">Can view</a></li>
+                                                    <li {% if not is_edit_user and not is_owner_user %}class="disabled"{% endif %}><a data-role="edit">Can edit</a></li>
+                                                    <li {% if not is_owner_user %}class="disabled"{% endif %}><a data-role="owner">Is owner</a></li>
```

Comment:
@mauriel133 On second thought, I remembered we had some discussions earlier on exposing "Manage access" button to users who do not have edit permissions so that they can grant view permissions to other users. Although the checks for view and edit are not needed now, they may be needed in the future, so I think it is good to keep these checks as is now.